src/ui/toCSSLineSpacing.js

```diff
@@ -25,8 +27,26 @@ export default function toCSSLineSpacing(source: any): string {
     strValue = String(Math.round(numValue * 100)) + '%';
   }
 
-  if (DEPRECATED_SINGLE_LINE_SPACING_VALUES.has(strValue)) {
```

Comment:
this used to handle 100% as well, do we still need a case for that?

Reply:
I think so. I'll add them back.